addons/l10n_tr/data/template/account.tax-tr.csv

```diff
@@ -1,17 +1,112 @@
+"","","","","","","","","","","","tax","invoice","tr391005","-PUR_20_TAX_10_10",""
+"","","","","","","","","","","","base","refund","","-PUR_20_BASE_10_10",""
+"","","","","","","","","","","","tax","refund","tr391005","+PUR_20_TAX_10_10",""
+"tr_p_vat_wh_20_2_10","23","Purchase VAT with Withholding Tax 20% (2/10)","","WH 20% (2/10)","False","","group","tr_p_wh_20_2_10,tr_p_20","purchase","tr_tax_group_kdv_20","","","","","Stopajlı Satın Alma KDV'si %20 (2/10)"
+"tr_p_vat_wh_20_5_10","24","Purchase VAT with Withholding Tax 20% (5/10)","","WH 20% (5/10)","False","","group","tr_p_wh_20_5_10,tr_p_20","purchase","tr_tax_group_kdv_20","","","","","Stopajlı Satın Alma KDV'si %20 (5/10)"
+"tr_p_vat_wh_20_7_10","25","Purchase VAT with Withholding Tax 20% (7/10)","","WH 20% (7/10)","False","","group","tr_p_wh_20_7_10,tr_p_20","purchase","tr_tax_group_kdv_20","","","","","Stopajlı Satın Alma KDV'si %20 (7/10)"
+"tr_p_vat_wh_20_9_10","26","Purchase VAT with Withholding Tax 20% (9/10)","","WH 20% (9/10)","False","","group","tr_p_wh_20_9_10,tr_p_20","purchase","tr_tax_group_kdv_20","","","","","Stopajlı Satın Alma KDV'si %20 (9/10)"
+"tr_p_vat_wh_20_10_10","27","Purchase VAT with Withholding Tax 20% (10/10)","","WH 20% (10/10)","False","","group","tr_p_wh_20_10_10,tr_p_20","purchase","tr_tax_group_kdv_20","","","","","Stopajlı Satın Alma KDV'si %20 (10/10)"
+"tr_p_wh_20_3_10","28","Purchasing Withheld VAT 20% (3/10)","-6%","WH 20% (3/10)","False","-6","percent","","none","tr_tax_group_wh_6","base","invoice","","+PUR_20_BASE_3_10","Satın Almada Kesilen KDV %20 (3/10)"
+"","","","","","","","","","","","tax","invoice","tr391005","-PUR_20_TAX_3_10",""
+"","","","","","","","","","","","base","refund","","-PUR_20_BASE_3_10",""
+"","","","","","","","","","","","tax","refund","tr391005","+PUR_20_TAX_3_10",""
+"tr_p_wh_20_4_10","29","Purchasing Withheld VAT 20% (4/10)","-8%","WH 20% (4/10)","False","-8","percent","","none","tr_tax_group_wh_8","base","invoice","","+PUR_20_BASE_4_10","Satın Almada Kesilen KDV %20 (4/10)"
+"","","","","","","","","","","","tax","invoice","tr391005","-PUR_20_TAX_4_10",""
+"","","","","","","","","","","","base","refund","","-PUR_20_BASE_4_10",""
+"","","","","","","","","","","","tax","refund","tr391005","+PUR_20_TAX_4_10",""
+"tr_p_vat_wh_20_3_10","30","Purchase VAT with Withholding Tax 20% (3/10)","","WH 20% (3/10)","False","","group","tr_p_wh_20_3_10,tr_p_20","purchase","tr_tax_group_kdv_20","","","","","Stopajlı Satın Alma KDV'si %20 (3/10)"
+"tr_p_vat_wh_20_4_10","31","Purchase VAT with Withholding Tax 20% (4/10)","","WH 20% (4/10)","False","","group","tr_p_wh_20_4_10,tr_p_20","purchase","tr_tax_group_kdv_20","","","","","Stopajlı Satın Alma KDV'si %20 (4/10)"
+"tr_s_20","32","Sales VAT 20%","20%","VAT 20%","False","20","percent","","sale","tr_tax_group_kdv_20","base","invoice","","+SAL_BASE_20","Satış KDV'si %20"

```

Comment:
```suggestion
"tr_s_20","32","Sales VAT 20%","20%","20%","False","20","percent","","sale","tr_tax_group_kdv_20","base","invoice","","+SAL_BASE_20","Satış KDV'si %20"

```
No need to specify VAT in the name, just the percentage is fine. Same for the ones below